.github/workflows/ci.yml

```diff
@@ -224,9 +233,13 @@ jobs:
         run: |
           mkdir app-build
           cd app-build
-          cmake ../app -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} -DCMAKE_CXX_FLAGS="${{ matrix.cxx_flags }}"
+          if [[ "${{ matrix.build }}" =~ ^(coverage)$ ]]; then
+            cmake ../app -DCODE_COVERAGE=ON -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} -DCMAKE_CXX_FLAGS="${{ matrix.cxx_flags }}"
+          else
+            cmake ../app -DCMAKE_BUILD_TYPE=${{ matrix.build_type }} -DCMAKE_CXX_FLAGS="${{ matrix.cxx_flags }}"
```

Comment:
@eseiler The only difference is `-DCODE_COVERAGE=ON`, can't we just add a `${{ matrix.additional_cmake_flags }}` ?

Reply:
`-DCODE_COVERAGE=${{ matrix.build_type == 'Coverage' && 'ON' || 'OFF' }}` should probably work (maybe setting it as env variable for this step is cleaner).

I would do this in my follow-up PR, because I probably change something anyways